packages/react/src/banner-description/BannerDescription.tsx

```diff
@@ -0,0 +1,19 @@
+import { type ComponentPropsWithRef, forwardRef } from "react";
+
+import { Box } from "../box";
+import { Flex } from "../flex";
+
+type AlertDescriptionProps = ComponentPropsWithRef<typeof Box>;
```

Comment:
Lets rename props to BannerDescriptionProps
```suggestion
type BannerDescriptionProps = ComponentPropsWithRef<typeof Box>;
```